packages/worker/src/utilities/email.ts

```diff
@@ -24,6 +25,13 @@ type SendEmailOpts = {
   cc?: boolean
   bcc?: boolean
   automation?: boolean
+  invite?: {
```

Comment:
If we break this out into its own type we could move it into `types` and make it available for the server to use.

Reply:
EmailInvite type added